### ISSUE TITLE
Topic Group Session Ideas

### ISSUE THREAD

**Issue by vantuyls** [vantuyls/studyGroup#8]
Hi All, use this space to suggest 1-2 hour Study Group sessions focused on a specific topic or tool. Some suggestions based on what i heard at our first Hacky Hour include:
- [ ] mapbox
- [ ] any R package you like
- [ ] any python library you like
- [ ] intro to git
- [ ] open science tools such as Open Science Framework
- [ ] code review
- [ ] informal coding session (workalong)

If you're interested in leading a session for one of these topics or for another topic you're interested in, let us all know here!

**Comment by wallacez** [comment]
I would be interested in the following options listed above:
- any R package you like
- any python library you like
- open science tools such as Open Science Framework

**Comment by oneilsh** [comment]
Ah I was hoping to make it to the hacky hour but ended up with a conflict :( 

Is the interest primarily in the scientific/practical realm? I could cover some fun stuff (I'm a big fan of "creative coding" libraries like processing.org and p5.js).

On the practical side, can also do some R things (e.g. using S3 classes, or writing packages as a more advanced followup, or even "R from a CS perspective") or Python stuff (maybe very basic GUIs with Tkinter library).

**Comment by wallacez** [comment]
Creative coding libraries could be fun as well.  Sometimes I need a break from my data but still want to code.

**Comment by grovduck** [comment]
I like the "creative coding" idea ... I'd show up to that.

I've got beginning to intermediate experience using Mapbox and a number of python libraries (numpy, pandas, rasterio), so would be happy to help lead/take part in one of those sessions.  We also had a cool little MapTime group in Corvallis that went away because the lead organizer (Nick Martinelli) left town.  But we were looking as JS mapping libraries as well as working with OpenStreetMap data

**Comment by vantuyls** [comment]
Yeah, @oneilsh, we're not too focused on serious or scientific. Creative and fun should definitely be part of what this is about.

**Comment by vantuyls** [comment]
All: i'd like to set up two topic-based study group sessions. One on Open Science Framework (which i'll lead) and one on Mapbox. @grovduck any chance you could lead a 1.5 hour session on Mapbox? I'm looking to set these up for the first two weeks in April. 

Then another social hacky hour at the end of April?

s

**Comment by grovduck** [comment]
@vantuyls, I'd love to do this, but I'm going to be slammed for the first week in April and gone the second week.  I'd be happy to pick this up for May if there still is interest.

**Comment by vantuyls** [comment]
@grovduck @oneilsh can i schedule up some sessions with you as leads (beginner mapbox and one of those creative coding sessions, respectively)? thinking these would be 1-2 hour sessions (up to you) sometime last week of april through mid may? I'll also run an Open Science Framework session in there somewhere.

I can find meeting rooms in the library. lunch sessions work for you?

s

**Comment by oneilsh** [comment]
Hi @vantuyls; sometime after Apr 29 would work best for me, as I might have some material I can bring by then. Any idea what the audience might be like skill-level wise?

**Comment by grovduck** [comment]
@vantuyls, yep I agree with @oneilsh that sometime in early May would work best for me (crazy April ...).  I could do a 1.0 to 1.5 hour intro on Mapbox styling.  Would attendees have their own computers or more seminar style?

**Comment by vantuyls** [comment]
@grovduck @oneilsh can either of you do a session on May 2, May 9, or May 17 over the lunch hour? I can reserve a room in the Library. Let me know ASAP and i'll get you scheduled in. 

@oneilsh as for audience - maybe target somewhere in "the middle": not too introductory, but not expecting experts. Talking with folks at the Hacky Hour, it seemed like many/most of the folks there had at least a working understanding of R/Python. You also may have some more experienced helpers in the room.

**Comment by grovduck** [comment]
@vantuyls, May 17 looks open for me right now and will give me plenty of time to put something together.

**Comment by oneilsh** [comment]
@vantuyls I can do lunch time (noon to 1) on the May 2 slot :) I'll talk about p5.js
